### PR TITLE
docs: added hacktoberfest instructions to main README.md

### PR DESCRIPTION
**Pull request by kodebae**
This PR added instructions for Hacktoberfest 23 to the main README.md. The purpose is twofold, to drum up interest in updating the WWCode Greenville docs, and to create much needed documentation for the chapter. Please take a look at the simple instructions and merge to main if you find the content to be appropriate. @klmansel please clarify if you have any specific feedback for me or additional instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +41,17 @@ We are still getting started with this project, but what we have so far will be
 <!-- GETTING STARTED -->
 ## Getting Started
 
+<!-- Hacktoberfest 23 -->
+## 🎃 Hacktoberfest 23
+
+- 🍴 To get started fork this repo
+- 👯‍♀️ Clone the repo to your laptop
+- 👩🏽‍💻 Participate by closing our open issues and contributing to the docs
+- 🤓 You must make 4 PR (pull requests) and have 4 MR (merge requests)
```
Comment (klmansel) [inline README.md:50]: this is actually the same thing @kodebae 
GitLab calls PRs merge requests

Comment (klmansel) [reply README.md:50]: Also, you can clarify that, ... _to get credit for participation you must_ ...

**Review by klmansel — APPROVED** [review]
Approved after two small changes! Thank you!!

**Comment by kodebae** [comment]
I have pushed the changes and updated the docs where indicated. 🙂

**Pull request merged**